action.yml

```diff
@@ -42,6 +50,8 @@ runs:
       shell: bash
     - run: |
         wokwi-cli --timeout "${{ inputs.timeout }}" --expect-text "$CI_EXPECT_TEXT" \
+          ${{ inputs.scenario != '' && format('--scenario "{0}"', inputs.scenario) }} \
```

Comment:
I guess this should be elf instead of scenario?

Reply:
Yeah, I don't know what was going on inside my head 😅